src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -237,25 +237,19 @@
     <xsd:complexType name="workflow">
         <xsd:sequence>
             <xsd:element name="marking-store" type="marking_store" />
-            <xsd:element name="supports" type="xsd:string" minOccurs="1" maxOccurs="unbounded" />
-            <xsd:element name="places" type="xsd:string" minOccurs="1" maxOccurs="unbounded" />
-            <xsd:element name="transitions" type="transitions" />
+            <xsd:element name="support" type="xsd:string" minOccurs="1" maxOccurs="unbounded" />
+            <xsd:element name="place" type="xsd:string" minOccurs="1" maxOccurs="unbounded" />
+            <xsd:element name="transition" type="transition" minOccurs="1" maxOccurs="unbounded" />
         </xsd:sequence>
         <xsd:attribute name="name" type="xsd:string" />
     </xsd:complexType>
 
     <xsd:complexType name="marking_store">
         <xsd:sequence>
-            <xsd:element name="type" type="xsd:string" minOccurs="0" maxOccurs="1" />
-            <xsd:element name="arguments" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
-            <xsd:element name="service" type="xsd:string" minOccurs="0" maxOccurs="1" />
-        </xsd:sequence>
-    </xsd:complexType>
-
-    <xsd:complexType name="transitions">
-        <xsd:sequence>
-            <xsd:element name="transition" type="transition" />
+            <xsd:element name="argument" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
         </xsd:sequence>
+        <xsd:attribute name="type" type="xsd:string" use="optional" />
+        <xsd:attribute name="service" type="xsd:string" use="optional" />
```

Comment:
You're right. I reverted this change and made the `name` attributes of the workflow and transition required instead.